docs/section19/19-11.md

```diff
@@ -0,0 +1,44 @@
+# Section 19 - Rule 19-11         
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** False    
+**Rule ID:** 19-11         
+**Rule Description:** For systems that serve computer rooms, if the  baseline system is HVAC System 11, it shall include an integrated fluid economizer meeting the requirements of Section 6.5.1.2 in the baseline building design.  
+
+**Rule Assertion:** Options NOT_APPLICABLE/UNDETERMINED      
+**Appendix G Section:** G3.1.2.6.1        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Baseline HVAC systems that serve computer rooms and are of system type 11.    
+ 
+
+**Function Calls:**  
+1. get_baseline_system_types  
+2. get_HVAC_systems_primarily_serving_comp_rooms  
+3. baseline_system_type_compare   
+
+
+## Rule Logic:   
+**Applicability Check 1 - Check if there are any Applicable Baseline System Types in the Project at All**   
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_11]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side system that is of type 11, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    This segment identifies applicable HVAC system types (outcome for inapplicable system types will be NOT_APPLICABLE)
+    - Get list of hvac systems in the B_RMI primarily serving computer rooms: `HVAC_systems_primarily_serving_comp_rooms_list = get_HVAC_systems_primarily_serving_comp_rooms(B_RMI)`  
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type and that it serves computer rooms, if yes then carry on, if no then NOT_APPLICABLE outcome for the hvac system:  `If baseline_system_type_compare(sys_type, HVAC_SYS.SYS_11, false) == TRUE AND hvac.id in HVAC_systems_primarily_serving_comp_rooms_list: `     
+                
+            - **Rule Assertion:** 
+            - Case 1: The system primarily serves computer rooms and the baseline system type is 11 then outcome is UNDETERMINED: `outcome = "UNDETERMINED" and raise_message "<Insert hvac.id> serves computer room(s) and is of baseline system type 11, conduct a manual check that an integrated fluid economizer meeting the requirements of Section 6.5.1.2 was modeled in the baseline building design."`  
```

Comment:
Do we want to add field to the schema (Loop or chiller object?) about fluid economizer? The we could take this RDS one step further, to verify whether economizer is specified, so that reviewer just needs to check whether it complies with 6.5.1.2. No strong feeling about it though.